src/Client/Model/Order.php

```diff
@@ -517,6 +519,18 @@ public function setBuyer(?OrderBuyer $buyer): Order
         return $this;
     }
 
+
+    public function getWeeePickup(): ?bool
```

Comment:
@BrandtM weeePickup ist bereits implementiert + unittests

https://github.com/jtl-scx/channel-core/commit/cd74ca14676baebf6e628235f32d972f7438bb77

Wie kommt das hier plötzlich rein?

Reply:
Das weeePickup kommt durch das mergen vom master rein. Es ist erst jetzt in den Models, da vor diesem Ticket die Generierung der Models ja nicht funktionierte.